saleor/dashboard/product/forms.py

```diff
@@ -14,6 +16,20 @@
 from .widgets import ImagePreviewWidget
 
 
+class RichTextField(forms.CharField):
+    """
+    A field for rich text editor, providing backend sanitization
+    """
+
+    widget = RichTextEditorWidget
+    ALLOWED_TAGS.extend([u'br', u'p', u'h2', u'h3'])
```

Comment:
Strings are unicode by default in Python 3, you don't have to wrap them with `u`.

Reply:
Also, we shouldn't change the global `ALLOWED_TAGS` but create a local copy and customize it. I think we could keep ALLOWED_TAGS in RichTextField, so we should rename the original import:
```
from bleach.sanitizer import ALLOWED_TAGS as BLEACH_ALLOWED_TAGS

# in RichTextField
ALLOWED_TAGS = BLEACH_ALLOWED_TAGS + ["..."]
```